### ISSUE TITLE
La aplicación crashea en modo apaisado

### ISSUE THREAD

**Issue by mx-psi** [fjmpiq/NPI#23]
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: conversandroid.talkback, PID: 17525
    java.lang.RuntimeException: Unable to destroy activity {conversandroid.talkback/conversandroid.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.speech.SpeechRecognizer.stopListening()' on a null object reference
        at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3911)
        ...
D/AppTracker: App Event: crash
```
